jsapp/xlform/src/model.inputParser.coffee

```diff
@@ -100,7 +100,11 @@ module.exports = do ->
     translations = o.translations
     if translations
       if translations.length > 1 and translations.indexOf(null) isnt -1
-        throw new Error("There is an unnamed translation in your form definition. Please give a name to all translations in your form.")
+        throw new Error("""
+          There is an unnamed translation in your form definition.
+          Please give a name to all translations in your form.
+          Use "Manage Translations" modal accessible from form landing page.
```

Comment:
Regular users won't know what a "modal" is. The term is too technical. 

But, anyhow, in the future, there should be a link displayed that loads the Translations modal window. Or a button in the header of the form builder.

Reply:
Changed it on the main branch to `Use "Manage Translations" option from form landing page.`